### PR TITLE
Add support for fdb flush in sairedis

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

**Comment by kcudnik** [comment]
test this please

File: meta/sai_meta.cpp [hunk 1]
```diff
@@ -4737,3 +4737,154 @@ void meta_sai_on_fdb_event(
...
+    SWSS_LOG_ENTER();
+
+    if (flush_fdb_entries == NULL)
+    {
+        SWSS_LOG_ERROR("function pointer is NULL");
+
+        return SAI_STATUS_INVALID_PARAMETER;
+    }
+
+    if (attr_count > MAX_LIST_COUNT)
```
Comment (keboliu) [inline meta/sai_meta.cpp]: should at least have one attribute in the list?

**Comment by keboliu** [comment]
I tested this with my PRs, it works well.

File: meta/sai_meta.cpp (hunk 1 above)
Comment (kcudnik) [reply meta/sai_meta.cpp]: no, since list can be empty

**Comment by kcudnik** [comment]
test this please

**Pull request merged**